README.md

```diff
@@ -14,7 +38,48 @@ At the core of Level are simple key-value stores that follow the characteristics
 
 To bring those concepts to Node.js and other JavaScript runtimes, Level utilizes idiomatic Node.js interfaces like [streams](https://nodejs.org/api/stream.html), [events](https://nodejs.org/api/events.html) and [buffers](https://nodejs.org/api/buffer.html). It offers a rich set of data types through [encodings](https://github.com/Level/encoding-down) and allows for extensions like [`subleveldown`](https://github.com/Level/subleveldown) to split a database into evented sections. Underlying stores can be easily swapped to target a wide range of runtime environments. The most common store is [`leveldown`](https://github.com/Level/leveldown) which is a pure C++ binding to LevelDB. [Many alternatives are available](https://github.com/Level/awesome/#stores) such as [`level-js`](https://github.com/Level/level-js) in the browser or [`memdown`](https://github.com/Level/memdown) for an in-memory store.
 
-The [`level`](https://github.com/Level/level) module is the recommended way to get started. Visit [`awesome`](https://github.com/Level/awesome) to discover more modules. See our [project board](https://github.com/orgs/Level/projects/2) to find out what we're working on. If you need help - technical, philosophical or other - feel free to open an issue in [`community`](https://github.com/Level/community) or a more specific repository.
+## FAQ
+
+### Where do I start?
+
+The [`level`](https://github.com/Level/level) module is the recommended way to get started. It offers a persistent database that works in Node.js and browsers. To store data in a different way, you might like:
+
+- `level-mem` to keep data in-memory (Node.js and browsers)
+- `level-rocksdb` to use RocksDB instead of LevelDB (Node.js only).
```

Comment:
Do we want to feature rocksdb like this? Is it the 2nd stable integration?

Reply:
Yeah, maybe not. It's stable, but less popular and often lagging in maintenance.

I'll rephrase the second paragraph to explain that there are alternatives to `level`, using only `level-mem` as an example